README.md

```diff
@@ -1,54 +1,99 @@
+### Uninstall
 
-## Creating a native executable
+To delete a deployed *notification-service*, use the file `deploy/*.yaml` files from this repository and delete the Kubernetes resources:
 
-You can create a native executable using: 
-```shell script
-./mvnw package -Pnative
+```console
+kubectl delete -f deploy/service.yaml
 ```
 
-Or, if you don't have GraalVM installed, you can run the native executable build in a container using: 
-```shell script
-./mvnw package -Pnative -Dquarkus.native.container-build=true
+## Usage
+
+The goal of the *notification-service* is to inform the user about a new event via a notification in a chat app. (currently Slack but extendable)
+
+It needs the following configuration:
+
+### Kubernetes secrets
+
+* for Slack Environment
+
+  The Kubernetes secret `slack-access` containing the values `SLACK_TOKEN` and `SLACK_CHANNEL` is needed.
+
+* for Keptn Environment
+
+  The Kubernetes secret `keptn-access` containing the value `KEPTN_BRIDGE_DOMAIN` is needed.
+
+These variables can be set in the `service.yaml` file, but their name must not be changed!
+```yaml
+- name: SLACK_TOKEN
+  valueFrom:
+    secretKeyRef:
+      key: SLACK_TOKEN
+      name: slack-access
+- name: SLACK_CHANNEL
+  valueFrom:
+    secretKeyRef:
+      key: SLACK_CHANNEL
+      name: slack-access
+- name: KEPTN_BRIDGE_DOMAIN
+  valueFrom:
+    secretKeyRef:
+      key: KEPTN_BRIDGE_DOMAIN
+      name: keptn-access
 ```
 
-You can then execute your native executable with: `./target/quarkus-service-1.0.0-SNAPSHOT-runner`
+<!-- is an installation like https://github.com/keptn-sandbox/monaco-service/blob/main/README.md#option-1-monaco-projects-folders needed? -->
+
+## Development
+
+This is an open source project, so I welcome any contributions to make it even better!
+
+### Build yourself
+
+* Build the binary: `./mvnw package -Pnative` (with GraalVM) `./mvnw package -Pnative -Dquarkus.native.container-build=true` (without GraalVM)
+* Build not a binary: `./mvnw package`
+* Run tests: ``
+
+<!-- what name should the docker image have / the value before the '/' -->
+* Build the docker image: `docker build -f src/main/docker/Dockerfile.native -t quarkus/notification-service .`
+* Run the docker image locally: `docker run -i --rm -p 8080:8080 quarkus/notification-service`
 
-If you want to learn more about building native executables, please consult https://quarkus.io/guides/maven-tooling.html.
+<!-- Also true for my project? -->
+* Push the docker image to DockerHub: `docker push keptnsandbox/notification-service:dev`
+* Deploy the service using `kubectl`: `kubectl apply -f deploy/`
+* Delete/undeploy the service using `kubectl`: `kubectl delete -f deploy/`
+* Watch the deployment using `kubectl`: `kubectl -n keptn get deployment notification-service -o wide`
 
-## Provided Code
+<!-- error: a container name must be specified for pod ... What should I do? -->
+* Get logs using `kubectl`: `kubectl -n keptn logs deployment/notification-service -f`
```

Comment:
In this case, the container name of the service must be specified as well (since we have multiple containers in one pod): `kubectl -n keptn logs deployment/notification-service -c notification-service- f`